mail_composer_cc_bcc/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Email CC and BCC",
+    "summary": "This module enables sending mail to CC and BCC partners in mail composer form.",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Social",
+    "website": "https://github.com/OCA/social",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Camptocamp, Odoo Community Association (OCA)",
```
can you pls do this change in another commit that can be backported?

`mail_compose_cc_bcc: fix authorship`

Thanks!

Reply:
this is missing